### PR TITLE
Prepare release/4.0.0 rc3

### PR DESCRIPTION
**Pull request by drgora**
This PR updates release notes, changelogs, and checkpoints for zend 4.0.0-rc3 release

### CODE REVIEW

File: src/chainparams.cpp
Comment (JackPiri) [inline src/chainparams.cpp]: Since we want this 4.0.0-rc3 to add only one checkpoint to previous 3.3.1, the checkpoint with height 1328300 should be dropped (it was introduced with 4.0.0-rc2).

**Review by JackPiri — CHANGES_REQUESTED** [review]
The PR is fine to me, please update `chainparams.cpp` as per my comment before getting final approval.
Thanks

File: src/chainparams.cpp
Comment (drgora) [reply src/chainparams.cpp]: Makes total sense, applied your suggestion, thanks

**Review by JackPiri — APPROVED** [review]

**Review by a-petrini — APPROVED** [review]
This PR looks fine to me!

**Review by ptagl — APPROVED** [review]

File: contrib/debian/changelog
Comment (cronicc) [inline contrib/debian/changelog]:
```suggestion
zen (4.0.0-rc3) stable; urgency=medium
```
See ZEND-426 on Jira.

**Review by cronicc — CHANGES_REQUESTED** [review]

File: contrib/debian/changelog
Comment (drgora) [reply contrib/debian/changelog]: Updated accordingly, thanks

**Pull request merged**
